src/config/CodeGeneratorConfig.php

```diff
@@ -4,9 +4,9 @@
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
 class CodeGeneratorConfig {
-	
+
 	protected $options;
-	
+
 	public function __construct(array $options = []) {
```

Comment:
Yes, now you should never really set a return type for them so its not really an issue.